src/software/ai/hl/stp/tactic/kick/kick_fsm.h

```diff
@@ -45,6 +56,13 @@ struct KickFSM
                 processEvent(GetBehindBallFSM::Update(control_params, event.common));
             };
 
+        /**
+         * Guard that checks if the ball has been chicked
```

Comment:
is the word "chicked" just the combination of "chipped" and "kicked"?

Reply:
yes